ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html

```diff
@@ -81,7 +81,7 @@ <h2 class="section-title" translate>HomePage.StepsSection.Title</h2>
                     </div>
                     <div>
                         <p translate>HomePage.StepsSection.Step4.Text</p>
-                        <a routerLink="/participation" class="button button_medium button_secondary" translate>
+                        <a href="/participation" class="button button_medium button_secondary" translate>
```

Comment:
It should be `routerLink` instead of `href`, shouldn't it?

Reply:
nah my bad i'll correct it

Reply:
it's done